common.h

```diff
@@ -491,6 +493,7 @@ namespace physicalconstants {
    const Real MASS_ELECTRON = 9.10938188e-31; /**< Electron rest mass, units: kg.*/
    const Real MASS_PROTON = 1.67262158e-27; /*!< Proton rest mass, units: kg.*/
    const Real R_E = 6.3712e6; /*!< radius of the Earth, units: m. */
+   const Real g = 9.80665; /*!< Standard acceleration of gravity at Earth's surface, units: m s^-2.*/
```

Comment:
Taken from WIkipedia I think – the digits here will never be critical in this code...

Reply:
My fist thought when I saw this was that it would be the quantum mechanical [g-factor](https://en.wikipedia.org/wiki/G-factor_(physics)) instead of Earth's surface gravity. Should we rename this to "EarthGravity", just to avoid confusion?